### PR TITLE
add more explicit instructions for terminal commands and adding people to repos in the encryption chapter

### PR DESCRIPTION
**Pull request by collinschwantes**
- provides more code snippets for running things in the command line
- adds a section on adding contributors to a git-crypt repo
- emphasizes adding the gpg tty variable to your settings file and explains how to do that more explicitly

### CODE REVIEW

File: encryption.Rmd [hunk 1]
```diff
@@ -85,6 +94,26 @@ your Keybase keys to use.  Instruction are found [here](https://blog.scottlowe.o
...
+having to run the command when you use git-crypt or sign commits. Use your favorite text editor (nano, vim, etc) to modify the settings file:
+
+```
+# on a mac in terminal
+nano .zshrc
+
+# in the nano editor
+
+export GPG_TTY=$(tty)
+
```
Comment (mekaczmarek) [inline encryption.Rmd:113]: I would add how to quit nano for those of us who don't use the terminal regularly.

File: encryption.Rmd [hunk 2]
```diff
@@ -43,7 +43,16 @@ command line tools.
...
+them on a mac, use [`homebrew`](https://docs.brew.sh/Installation). You should also install `pinentry`.
+
+```
+# In Terminal 
+## homebrew automatically updates when you run it so if you haven't used it in a
+## while there may be a somewhat lengthy update
+
+brew install gpg
+brew install git-crypt
+```
```
Comment (mekaczmarek) [inline encryption.Rmd:55]: Maybe add brew install pinentry. Just to be super explicit, even though it may be overkill.

File: encryption.Rmd [hunk 3]
```diff
@@ -85,6 +94,26 @@ your Keybase keys to use.  Instruction are found [here](https://blog.scottlowe.o
 your keys to maximum trust level.
 
 
+### Set the text entry for gpg 
+
+A common source of errors is that the text entry for `gpg` isn't set properly. You can fix this by setting the `GPG_TTY` environment variable like so:
```
Comment (mekaczmarek) [inline encryption.Rmd:99]: Clarify what "text entry" means. Again, this is for the computationally challenged/newbies.

**Review by mekaczmarek — COMMENTED** [review]
Looks good to me! I may add some resources on encryption for those who want to learn more about how it works. If this is beyond the scope of the manual, then feel free to ignore!

File: encryption.Rmd [hunk 4]
```diff
@@ -85,6 +95,29 @@ your Keybase keys to use.  Instruction are found [here](https://blog.scottlowe.o
...
+
+```
+# on a mac in terminal
+nano .zshrc
+
+# in the nano editor
+
+export GPG_TTY=$(tty)
+
+# press control o to write, then follow prompts to save
```
Comment (noamross) [inline encryption.Rmd:115]:
```suggestion
# press Ctrl-O to write ("write-Out"), then follow prompts to save
```

File: encryption.Rmd [hunk 5]
```diff
@@ -85,6 +95,29 @@ your Keybase keys to use.  Instruction are found [here](https://blog.scottlowe.o
...
+# press control x to exit
```
Comment (noamross) [inline encryption.Rmd:116]:
```suggestion
# press Ctrl-X to exit
```

**Review by noamross — APPROVED** [review]

**Review by n8layman — APPROVED** [review]
Do we have verbage that this should be .bashrc instead of .zshrc if using Linux? Otherwise looks great!

**Pull request merged**